<ui version="4.0" >
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" colspan="2" >
    <widget class="QSpinBox" name="moveSpinBox" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="buttonSymbols" >
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix" >
      <string/>
     </property>
     <property name="prefix" >
      <string>Move </string>
     </property>
     <property name="maximum" >
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="KLineEdit" name="playerLabel" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="capturesBox" >
     <property name="title" >
      <string>White</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="5" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Captures:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLabel" name="whiteCapturesLabel" >
        <property name="text" >
         <string>0</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Time left:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLabel" name="whiteTimeLabel" >
        <property name="text" >
         <string>00:00</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="KLineEdit" name="whitePlayerName" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Name</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="urlDropsEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="Line" name="line_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Black</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Captures:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="blackCapturesLabel" >
        <property name="text" >
         <string>0</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Time left:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="blackTimeLabel" >
        <property name="text" >
         <string>00:00</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="KLineEdit" name="blackPlayerName" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Computer (Level 10)</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="urlDropsEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="Line" name="line_4" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Komi:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>komiSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QDoubleSpinBox" name="komiSpinBox" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="buttonSymbols" >
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix" >
      <string> Points</string>
     </property>
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>50.000000000000000</double>
     </property>
     <property name="value" >
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Handicap:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>handicapSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QSpinBox" name="handicapSpinBox" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="buttonSymbols" >
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix" >
      <string> Stones</string>
     </property>
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>9</number>
     </property>
     <property name="value" >
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
